src/test/java/net/minecraftforge/debug/client/AudioStreamTest.java

```diff
@@ -0,0 +1,91 @@
+package net.minecraftforge.debug.client;
```

Comment:
Missing license header, please run the `updateLicenses`/`licenseFormat` task.

Reply:
Have updated the license for this file. Worth noting the year is outdated on all files, so by default running `updateLicenses` will touch 600+ files - probably worth that being run separately and committed to trunk.